<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">#minimizeButton {
  image: url(:/img/res/minimize-button1.png);
}

#minimizeButton, #maximizeButton {
  background-color: transparent;
  border-radius: 0px;
}

#minimizeButton:hover, #maximizeButton:hover {
  border-style: none;
  background-color: #80c7c7c7;
}

#minimizeButton:pressed, #maximizeButton:pressed {
  border-style: none;
  background-color: #80808080;
}

#closeButton {
  image: url(:/img/res/close-button1.png);
  background-color: transparent;
  border-radius: 0px;
}

#closeButton:hover {
	image: url(:/img/res/close-button2.png);
  border-style: none;
  background-color: #e81123;
}

#closeButton:pressed {
  image: url(:/img/res/close-button2.png);
  border-style: none;
  background-color: #8c0a15;
}

#fileBtn{
font: 9t &quot;华康方圆体W7(P)&quot;;
border-radius: 10px;
border-color: rgb(0, 170, 255);
background-color: rgb(170, 255, 255);
}

#fileBtn:pressed {

	background-color: rgb(170, 255, 255);
}
#startBtn{
font: 10t &quot;华康方圆体W7(P)&quot;;
border-radius: 10px;
border-color: rgb(0, 170, 255);
background-color: rgb(170, 255, 255);
}
#startBtn:pressed {
	background-color: rgb(170, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="ico">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/img/res/dog.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;华康方圆体W7(P)&quot;;</string>
       </property>
       <property name="text">
        <string>法雷序列</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>297</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="minimizeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Minimize</string>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maximizeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Maximize</string>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>217</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>246</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="FirstDepth">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
       </property>
       <property name="text">
        <string>深度优先</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FirstBreadth">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
       </property>
       <property name="text">
        <string>广度优先</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Iteration">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
       </property>
       <property name="text">
        <string>循环迭代</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item row="0" column="2">
          <widget class="QLineEdit" name="num"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="numLab">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;华康方圆体W7(P)&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="fileBtn">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;华康方圆体W7(P)&quot;;
border-radius: 10px;</string>
           </property>
           <property name="text">
            <string>选择保存的文件</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="fileName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;华康方圆体W7(P)&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startBtn">
           <property name="styleSheet">
            <string notr="true">font: 25 20pt &quot;华康方圆体W7(P)&quot;;</string>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="time1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>246</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fareylist.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimizeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
